ORION/scripts/test-prow-step.sh

```
@@ -0,0 +1,162 @@
+        export ES_SERVER="http://${ES_HOST}"
+    fi
+fi
+
+# Ensure /secret/perfscale-prod points to our mock secrets
+if [[ ! -e "/secret/perfscale-prod" ]] || [[ "$(readlink -f /secret/perfscale-prod 2>/dev/null)" != "$(readlink -f "$SECRET_DIR")" ]]; then
+    if [[ $EUID -eq 0 ]]; then
+        mkdir -p /secret
+        ln -sfn "$SECRET_DIR" /secret/perfscale-prod
+        echo "Created symlink /secret/perfscale-prod -> $SECRET_DIR"
+    else
+        read -p "Create /secret/perfscale-prod symlink with sudo? (y/n): " CREATE_SYMLINK
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)